### ISSUE TITLE
"Read KoJak's Story" link on homepage cannot be hovered or clicked

### ISSUE THREAD

**Issue by MichaelAllenWarner** [utkwdn/utk-atlas#40]
This is a `z-index` issue that I believe is caused by misuse of Bootstrap's [`stretched-link`](https://getbootstrap.com/docs/4.3/utilities/stretched-link/) utility class. The class is supposed to go on the actual link, but it's currently on the containing div instead:

<img width="371" alt="Screen Shot 2022-07-08 at 9 17 38 AM" src="https://user-images.githubusercontent.com/44578578/177999594-1c6bc28c-02ae-46e6-a331-f66a3851e664.png">

Demonstration of the problem:

https://user-images.githubusercontent.com/44578578/177998802-9118d710-974a-45cc-b54e-042b7a4cde05.mov

**Comment by spworky** [comment]
@pikemurdy  @MichaelAllenWarner The issue here is Wordpress doesn't add the stretched-link class as expected. If you select an element and use the advanced custom css class – it will add the class to a nested <a> tag – which makes the stretched-link class not function as intended by bootstrap (since it must be added as a class to a child of the main component you wish to have be clickable). To fix the functionality for now, I have just removed the stretched-link class allowing the button to function. 

The only inelegant solution I have at the moment is to add a html block with <a class="stretched-link" > as a direct child of the main card. In this particular instance, I am not sure we are gaining anything by having the whole card be clickable – but can add if we want that behavior. [I also did try just adding position-relative to the card-body (based on bootstrap doumentation), but since the .stretched-link is nested, it did not do the trick either.]
